src/configure/configure.ts

```diff
@@ -533,6 +535,16 @@ class Orchestrator {
         }
     }
 
+    private filterPipelinesByResourceType(pipelineMap: Map<string, PipelineTemplate[]>) {
+        if (this.inputs.targetResource.resource && this.inputs.targetResource.resource.id) {
+            pipelineMap.forEach((element) => {
+                if (element.length > 0 && element[0].targetType !== this.inputs.targetResource.resource.type) {
+                    pipelineMap.delete(element[0].label);
```

Comment:
why check for `element.length > 0`. can that even happen?

Reply:
Not required